data-prepper-plugins/mongodb/src/main/java/org/opensearch/dataprepper/plugins/mongo/stream/StreamWorker.java

```diff
@@ -72,6 +77,20 @@ public class StreamWorker {
     private final JsonWriterSettings writerSettings = JsonWriterSettings.builder()
             .outputMode(JsonMode.RELAXED)
             .objectIdConverter((value, writer) -> writer.writeString(value.toHexString()))
+            .binaryConverter((value, writer) ->  writer.writeString(Base64.getEncoder().encodeToString(value.getData())))
+            .dateTimeConverter((value, writer) -> writer.writeNumber(String.valueOf(value.longValue())))
```

Comment:
Similar to another comment - Can we avoid the string conversion?

```suggestion
            .dateTimeConverter((value, writer) -> writer.writeNumber(value.longValue()))
```

Reply:
`writer.writeNumber` only accepts String.